tests/Domain/ValueObject/PartialDateTest.php

```diff
@@ -92,4 +92,38 @@ public function invalidDateDataProvider()
             [[2015, 2, 29]],
         ];
     }
+
+    /**
+     * @dataProvider hasMonthDataProvider
+     */
+    public function testHasMonth($input, $expected)
+    {
+        $partialDate = new PartialDate(...$input);
+        $this->assertEquals($expected, $partialDate->hasMonth());
+    }
+
+    public function hasMonthDataProvider()
+    {
+        return [
+            [[2018, 0, 16], false],
```

Comment:
you know that you can label test data? 

'missing_month' => [....]
'nothing_missing' => [...]

it helps localising the test case failing in case someday it fails